package.json

```diff
@@ -48,13 +48,16 @@
     "jshint": "jshint --show-non-errors ."
   },
   "dependencies": {
-    "sax": "~1.2.1",
+    "cheerio": "^0.22.0",
     "coa": "~1.0.1",
-    "js-yaml": "~3.6.1",
     "colors": "~1.1.2",
-    "whet.extend": "~0.9.9",
+    "css": "^2.2.1",
```

Comment:
CSSO also has a parser (but probably with a bit different AST), so it could be reused without adding dependency.